sourcemash/frontend/templates/sidenav.tpl

```diff
@@ -41,10 +38,9 @@
           <div class="loading center-align"></div>
           <% feeds.where({subscribed: true}).forEach(function(feed) { %>
             <a href="/#feeds/<%= feed.get('id') %>"><li class="row collection-item">
-            <span class="col s7 truncate"><%= feed.get('title') %></span>
-            <span class="badge col s2"><%=feed.get('item_count')%></span>
-            <% if (feed.get('unread_count') > 0) { %>
-              <span class="new badge col s3"><%=feed.get('unread_count')%></span>
+            <span class="col s9 truncate"><%= feed.get('title') %></span>
+            <% if (feed.get('unread')) { %>
+              <span class="new badge col s3"></span>
```

Comment:
Instead of the heavy new label, what if we have a dot made with icon: mdi-image-brightness-1. We can color it whatever fits our theme and give it a tool tip that says "New Items"? "new" probably isn't descriptive enough because a user wouldn't know if that means it has new items or if it's a new category.